Scripts/RemediationScripts/Readme.md

```diff
@@ -1016,6 +1019,48 @@ Yes
 
 ___ 
 
+
+
+## 49. Azure_AVD_Audit_Enable_HostPool_BootDiagnostics
+
+### Display Name
+Boot Diagnostic must be enabled with Managed Storage Account on Azure AVD Host pool VMs.
+
+### Link to Bulk Remediation Script (BRS)
+[Remediate-BootDiagnosticsForAVDHotPool](Remediate-BootDiagnosticsForAVDHotPool.ps1)
```

Comment:
This link doesn't work in UI - gives 404

Reply:
Updated the text. Now working.